site/content/en/docs/administration/advanced/backup_guide.md

```diff
@@ -57,6 +57,11 @@ cvat_data.tar.bz2  cvat_db.tar.bz2  cvat_events.tar.bz2
 
 ## How to restore CVAT from backup
 
+__Warning: use exactly the same CVAT version to restore DB. Otherwise
+it will not work because between CVAT releases the layout of DB can be
+changed. You always can upgrade CVAT later. It will take care to migrate
+your data properly internally.__
+
```

Comment:
```suggestion
**Warning: use exactly the same CVAT version to restore DB. Otherwise
it will not work because between CVAT releases the layout of DB can be
changed. You always can upgrade CVAT later. It will take care to migrate
your data properly internally.**

```